01-node-tutorial/answers/content/practice2.js

```diff
@@ -0,0 +1,3 @@
+const sentence = "this is a sentence"
+
+exports.sentence = sentence
```

Comment:
This export isn't quite correct. Remember to use `module.exports` when exporting.

Reply:
Fixed with latest commit!